learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -15,70 +15,91 @@ redirect_from:
 
 ## Setting up the prerequisites
 
-To complete this tutorial, you need:
+>**Info:** You need a command terminal to complete this tutorial.
```

Comment:
Why we are differentiating this? Can't we list it as a prerequisite? To me, it is another prerequisite.

Reply:
## Prerequisites

To run this getting started tutorial, you need the following prerequisites:

1. [Ballerina 2202.0.0(Swan Lake)](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/) or greater
2. A text editor
   >**Tip:** Preferably, [Visual Studio Code](https://code.visualstudio.com/) with the [Ballerina extension](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) installed
3. A command terminal

Reply:
Done.